rdf/common/src/main/scala/org/w3/banana/io/AnyRDFReader.scala

```diff
@@ -0,0 +1,12 @@
+package org.w3.banana
+package io
+
+
+/** RDF reader. */
+trait AnyRDFReader[Rdf <: RDF, M[_]] {
+  
+	/** Read triples from the given location. The syntax is determined from input source URI
+   *  (content negotiation or extension). */
+   def readAnyRDFSyntax( uri:String) : M[Rdf#Graph]
```

Comment:
Why not ?  But this RDFReader with 2 parameters would be confusing with the original one with 3 parameters.